src/content/docs/index.mdx

```diff
@@ -20,6 +20,12 @@ hero:
 
 import ClassCardGrid from "../../components/ClassCardGrid.astro";
 
+## Upcoming Course Content
+
+import AbridgedSchedule from "../../components/AbridgedSchedule.astro";
+
+<AbridgedSchedule />
+
```

Comment:
It should not be _too_ hard to convert this code into something that will work in the `ClassCardGrid` and it will look way better as part of the splash page. [The starlight docs home page](https://starlight.astro.build/) have an example of what they look like rendered.